homepage/models.py

```diff
@@ -114,6 +115,17 @@ class AppUser(models.Model):
     def __str__(self):
         return self.user.username
 
+    @staticmethod
+    # Creates a new app_user (without superuser permmisions) and saves it in the DB
+    # The is_active attribute of AppUser.user is automaticly set to 'True' when creating a new AppUser
+    def create_app_user(username, email, password):
+        app_user = AppUser()
+        user = User.objects.create_user(username, email, password)
+        app_user.user = user
+

```

Comment:
please remove the blank line.
I would use a blank line as a "separator" if the method is too long and I have too many cases to separate them but in cases of a small method I think we don't have to add it.

Reply:
Removed